project-reports/2024/2024-annual-Hyperledger-Indy.md

```diff
@@ -0,0 +1,108 @@
+# 2024 Annual Review Hyperledger Indy
+
+---
+layout: default
+title: 2024 Annual Review Hyperledger Indy
+parent: 2024
+grand_parent: Project Updates
+---
+
+## Project Health
+
+Hyperledger Indy LFX Insights Page for [Calendar Year 2023](https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=indy&repository=all&dateFilters=2023-01-01%20to%202023-12-31&dateRange=2023-01-01%20to%202023-12-31&compare=PP&granularity=month&hideBots=true) (v. [2022](https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=indy&repository=all&dateFilters=2022-01-01%20to%202022-12-31&dateRange=2022-01-01%20to%202022-12-31&compare=PP&granularity=month&hideBots=true))  and for [Q1 2024 Oct-Dec.](https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=indy&repository=all&dateFilters=2023-10-01%20to%202023-12-31&dateRange=2023-10-01%20to%202023-12-31&compare=PP&granularity=month&hideBots=true) (v. [Q4 2023](https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=indy&repository=all&dateFilters=2023-07-01%20to%202023-09-30&dateRange=2023-07-01%20to%202023-09-30&compare=PP&granularity=month&hideBots=true)).
+
+Compared to 2022, the Pull Request count was down by about 25% to 189 for the year, while the number of contributors was slightly higher, up about 20% to 46. Compared to Q4 2023, the Q1 2024 report changes had the same ratios — down about 25% (to 24) in Pull Requests, up about 20% in contributors (to 13).
+
+While the existing implementation of Indy is extremely stable, and robust — for example, the Sovrin Foundation has been running its networks for 6 years, with 100% uptime since Feb. 2020 — there has been relatively little substantially new functionality added in some time. That may be changing as, in the latter half of 2023, the idea of “Indy Besu” was raised and a successful PoC developed, demonstrating implementing Indy’s public-permissioned network model on top of a Besu network. Progress has already been good in 2024 with the Indy community agreeing on the creation of the [Indy Besu repository](https://github.com/hyperledger/indy-besu) that further moves the implementation forward.
+
+Indy Besu is a set of independent smart contracts that **can be used for an existing Besu network** or built a new one from scratch. The main idea here is a modular system (modules list below) that allows deploying smart contracts at any time.
```

Comment:
Possible addition to clearly identify possible deployment options. I apologize for not immediately noting this point.
```suggestion
Indy Besu is a set of independent smart contracts. The main idea here is a modular system (modules list below) that allows deploying smart contracts at any time for 
* permissioned Indy netwrok (new or migration of exisitng ones)
* existing Besu networks
* existing or new Ethereum permissionless networks
```

Reply:
Updates made